manifests/pip/bootstrap.pp

```diff
@@ -19,26 +20,34 @@
       'AIX' => '/opt/freeware/bin',
       default => '/usr/bin'
     }
+
+    $environ = $facts['os']['family'] ? {
+      'AIX' => [ "http_proxy=${http_proxy}", "https_proxy=${http_proxy}" ],
+      default => [ "HTTP_PROXY=${http_proxy}", "HTTPS_PROXY=${http_proxy}" ]
+    }
+
     if $version == 'pip3' {
       exec { 'bootstrap pip3':
-        command => '/usr/bin/curl https://bootstrap.pypa.io/get-pip.py | python3',
-        unless  => 'which pip3',
-        path    => $python::params::pip_lookup_path,
-        require => Package['python3'],
+        command     => '/usr/bin/curl https://bootstrap.pypa.io/get-pip.py | python3',
+        environment => $environ,
+        unless      => 'which pip3',
+        path        => $python::params::pip_lookup_path,
+        require     => Package['python3'],
       }
       # puppet is opinionated about the pip command name
-      file { 'pip3-python':
+      -> file { 'pip3-python':
```

Comment:
please remove the arrow or the `require` parameter. Only one of it is needed.

Reply:
Done.